src/test-utils/browser-automation/browser-driver.js

```diff
@@ -58,30 +56,24 @@ export default class BrowserDriver {
     if (this.browser) {
       return Promise.resolve(this.browser);
     }
-    return this.puppeteer
-      .launch(options)
-      .then(browser => {
-        this.browser = browser;
-      })
-      .catch(error => {
-        console.error(addColor(ERR_AUTOMATION, COLOR.BRIGHT_RED)); // eslint-disable-line
-        throw error;
-      });
+    return this.puppeteer.launch(options).then(browser => {
```

Comment:
Nit: I personally prefer the one async action per line style, with each `then` on its own line.